app/assets/stylesheets/application.scss

```diff
@@ -312,10 +326,17 @@ main {
           display: inline-block;
         }
 
-        a {
+        button {
           @include core-16;
           padding: 0  5px 0 5px;
           display: inline-block;
+          -webkit-font-smoothing: antialiased;
+          -moz-osx-font-smoothing: grayscale;
+          font-size: 100%;
+          border-width: 0;
+          color: $govuk-blue;
+          background: none;
+          cursor: pointer;
```

Comment:
Is `cursor: pointer` really necessary on a button? I don't know of any browser that wouldn't do this by default.

Reply:
not necessary, as you could argue that pointer indicates a navigation action. by adding pointer i'm being consistent with https://design-system.service.gov.uk/components/accordion/ and https://govuk-publishing-components.herokuapp.com/component-guide/accordion